frontend-react/e2e/spec/all/idletimeout.spec.ts

```diff
@@ -4,10 +4,9 @@ import process from "node:process";
 import { OrganizationPage } from "../../pages/authenticated/admin/organization";
 import { test as baseTest } from "../../test";
 
-const timeout = parseInt(process.env.VITE_IDLE_TIMEOUT ?? "20000");
-// Add/Sub 500 ms to account for variance
-const timeoutLow = timeout - 500;
-const timeoutHigh = timeout + 500;
+const timeout = parseInt(process.env.VITE_IDLE_TIMEOUT ?? "900000");
```

Comment:
nit: can we make the `"900000"` an integer?

Reply:
Negative, the function expects a string so I'm making sure to provide a string fallback for the env var